custom_components/plugwise/__init__.py

```
@@ -143,3 +144,26 @@
+
+async def async_migrate_plugwise_entry(
+    hass: HomeAssistant,
+    coordinator: PlugwiseDataUpdateCoordinator,
+    entry: PlugwiseConfigEntry) -> bool:
+    """Migrate old config entry."""
+
+    _timeout = 30
+    if coordinator.api.smile_version >= "3.2.0":
+        _timeout = 10
+
+    if entry_version > 1:
```

Ruff

Undefined name `entry_version`

(F821)